.mill-version

```diff
@@ -0,0 +1 @@
+0.9.3
```

Comment:
I only added this file to make metals work better, I can remove it if you think

Reply:
If metals works better with the `.mill-version` file then we can keep it.